src/components/Footer.tsx

```diff
@@ -0,0 +1,35 @@
+import { useContext } from 'react';
+import { DispatchContext, StateContext } from '../Store';
+import { TodosFilter } from './TodosFilter';
+import { ActionType } from '../utils/enums';
+
+export const Footer = () => {
+  const dispatch = useContext(DispatchContext);
+  const { todos } = useContext(StateContext);
+
+  const completedTodos = todos.filter(todo => todo.completed === true);
+
```

Comment:
you can write instead

`const completedTodos = todos.filter(todo => todo.completed)`